packages/types/src/cowSwap.ts

```diff
@@ -163,7 +163,11 @@ export type OrderMetaData = {
   fullAppData?: string | null
 }
 
-export type Order = OrderCreation & OrderMetaData
+export type Order = OrderCreation &
+  OrderMetaData & {
+    // Not documented in API docs, but exists.
```

Comment:
![surprise](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExczh3dWIzb2VoaTZxbGF5ZWZ1cmVnNjlsdmQ0aGFmaG5rdmdjcDU5ayZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/bQtUQp1wRZIUujzJ7E/giphy.webp)

Disregard the previous CR comment about docs seemingly getting better on the CoW side of things